server/controllers/userController.js

```diff
@@ -50,4 +53,29 @@ const registerUser = asyncHandler(async (request, response) => {
   response.status(200).json({ message: "Success" });
 });
 
-export { authUser, registerUser };
+// @desc    Get user profile
+// @route   GET /api/users/profile
+// @access  Public
+//TODO: Work on authMiddleware.js then make private
+const getUserProfile = asyncHandler(async (request, response) => {
```

Comment:
Maybe we should add this function in a future ticket so it's not confusing when we work on it in the future..